lib/frameworks/mocha.js

```diff
@@ -18,7 +19,7 @@ module.exports.run = function(config, specs, capabilities) {
         requires = config.mochaOpts.require,
         runner;
 
-    if(typeof config.mochaOpts.ui !== 'string' || !config.mochaOpts.ui.match(/(bdd|tdd)/i)) {
+    if(typeof config.mochaOpts.ui !== 'string') {
```

Comment:
Mocha may have new interfaces added to it in the future. This sort of thing shouldn't be hard-coded.